### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MaksymMaliuk**
[DEMO LINK](https://MaksymMaliuk.github.io/react_person/)

### CODE REVIEW

**Review by Vlados3611 — APPROVED** [review]
Good work 👍

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,35 @@
...
+
+function hasPartner(isMaried, sex, partnerName) {
+  if (isMaried) {
+    return sex === 'm'
+      ? `${partnerName} is my wife`
+      : `${partnerName} is my husband`;
+  }
+
+  return 'I am not married';
+}
```
Comment (Andr1yK) [inline src/components/Person/Person.jsx:35]: avoid nested IFs, it's hard to reed :(

Comment (Andr1yK) [reply src/components/Person/Person.jsx:35]: let's refactor it)

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,35 @@
...
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
```
Comment (Andr1yK) [inline src/components/Person/Person.jsx:19]: fix indents

Comment (Andr1yK) [inline src/components/Person/Person.jsx:19]: if age = 0 I will see 0 :)

**Review by Andr1yK — CHANGES_REQUESTED** [review]

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**